modules/gapi/test/common/gapi_core_tests_inl.hpp

```diff
@@ -14,6 +14,327 @@
 namespace opencv_test
 {
 
+// FIXME avoid this code duplicate in perf tests
+namespace {
+void adjustBoundingBoxRef(cv::Rect& boundingBox) {
+    auto w = boundingBox.width;
+    auto h = boundingBox.height;
+
+    boundingBox.x -= static_cast<int>(0.067 * w);
+    boundingBox.y -= static_cast<int>(0.028 * h);
+
+    boundingBox.width += static_cast<int>(0.15 * w);
+    boundingBox.height += static_cast<int>(0.13 * h);
+
+    if (boundingBox.width < boundingBox.height) {
+        auto dx = (boundingBox.height - boundingBox.width);
+        boundingBox.x -= dx / 2;
+        boundingBox.width += dx;
+    } else {
+        auto dy = (boundingBox.width - boundingBox.height);
+        boundingBox.y -= dy / 2;
+        boundingBox.height += dy;
+    }
+}
+
+std::tuple<cv::Rect, float, float, float> unitedParseRef(const float* it,
+                                                         const cv::Size& in_size) {
+    float image_id   = it[0];
+    float label      = it[1];
+    float confidence = it[2];
+    float rc_left    = it[3];
+    float rc_top     = it[4];
+    float rc_right   = it[5];
+    float rc_bottom  = it[6];
+
+    cv::Rect rc;  // map relative coordinates to the original image scale
+    rc.x      = static_cast<int>(rc_left   * in_size.width);
+    rc.y      = static_cast<int>(rc_top    * in_size.height);
+    rc.width  = static_cast<int>(rc_right  * in_size.width)  - rc.x;
+    rc.height = static_cast<int>(rc_bottom * in_size.height) - rc.y;
+    return std::make_tuple(rc, image_id, confidence, label);
+}
+
+int randInRange(const int& start, const int& end)
+{
+    GAPI_Assert(start <= end);
+    return start + std::rand()%(end - start + 1);
```

Comment:
```suggestion
    return start + std::rand() % (end - start + 1);
```

Reply:
Consistency in arithmetic operators too (surrounded by spaces)

Reply:
Fixed.